pkg/kubelet/container_bridge.go

```diff
@@ -140,28 +142,17 @@ func cbr0CidrCorrect(wantCIDR *net.IPNet) bool {
 	return wantCIDR.IP.Equal(cbr0IP) && bytes.Equal(wantCIDR.Mask, cbr0CIDR.Mask)
 }
 
-// TODO(dawnchen): Using pkg/util/iptables
-// nonMasqueradeCIDR is the CIDR for our internal IP range; traffic to IPs outside this range will use IP masquerade.
-func ensureIPTablesMasqRule(nonMasqueradeCIDR string) error {
-	// Check if the MASQUERADE rule exist or not
-	if err := exec.Command("iptables",
-		"-t", "nat",
-		"-C", "POSTROUTING",
-		"!", "-d", nonMasqueradeCIDR,
+// nonMasqueradeCIDR is the CIDR for our internal IP range; traffic to IPs
+// outside this range will use IP masquerade.
+func ensureIPTablesMasqRule(client iptables.Interface, nonMasqueradeCIDR string) error {
+	if _, err := client.EnsureRule(iptables.Append, iptables.TableNAT,
```

Comment:
now that we're using util can you add a unittest for this? 
I leave it to you, since this code didn't have any to begin with, but I"ll wait for your nack.